Library/Homebrew/cmd/list.rb

```diff
@@ -36,6 +37,11 @@ class List < AbstractCommand
         switch "--pinned",
                description: "List only pinned formulae, or only the specified (pinned) " \
                             "formulae if <formula> are provided. See also `pin`, `unpin`."
+        switch "--manual", "--installed-on-request",
+               description: "List the formulae installed on request."
+        switch "--auto", "--installed-as-dependency",
+               description: "List the formulae installed automatically."
```

Comment:
```suggestion
        switch "--installed-on-request",
               description: "List the formulae installed on request."
        switch "--installed-as-dependency",
               description: "List the formulae installed automatically."
```

Feels nicer to me to be more consistent with the other uses. Could add a short flag to both here and e.g. `brew leaves` if desired, though.

Reply:
FYI the `-r` switch (and `-1`, `-l`, `-t`) are passed directly to `ls` when generating the `brew list` output, which is why those characters were selected.